### PR TITLE
Fully disable dependency generator in suse via undefine

### PR DESCRIPTION
**Pull request by watologo1**
and set
Group: Productivity/Networking/Boot/Servers
in SUSE case.

### CODE REVIEW

**Comment by SchoolGuy** [comment]
@watologo1 If finished, feel free to merge!

**Comment by Conan-Kudo** [comment]
For what it's worth, the generator doesn't need to be disabled in Tumbleweed anymore once this SR is merged: https://build.opensuse.org/request/show/792701

**Comment by SchoolGuy** [comment]
@Conan-Kudo Well all builds are failing... :/

**Comment by Conan-Kudo** [comment]
@SchoolGuy Just fixed that with https://build.opensuse.org/request/show/792707

**Comment by Conan-Kudo** [comment]
@SchoolGuy Blech, fixed with https://build.opensuse.org/request/show/792710

**Comment by watologo1** [comment]
Hi,

Thanks for the heads up.

Am Donnerstag, 9. April 2020, 14:02:39 CEST schrieb Neal Gompa (ニール・ゴンパ):
> @SchoolGuy Blech, fixed with https://build.opensuse.org/request/show/792710

I definitley want to get back to the generator.
But it must be newer than SLE 15 SP2.

This is not that urgent and letting some time run through certainly helps, 
since the generator is rather new in SUSE OBS rpm-python-macros.

Simply disabling and then add:

%if 0%{?suse_version} >= 15XX

around it is what  I want to head to in some week/months.

   Thomas

File: cobbler.spec [hunk 1]
```diff
@@ -37,6 +37,8 @@
 %define grub2_x64_efi_pkg grub2-x86_64-efi
 %define grub2_ia32_efi_pkg grub2-i386-efi
 %define system_release_pkg distribution-release
+%undefine python_enable_dependency_generator
+%undefine python_disable_dependency_generator
```
Comment (Conan-Kudo) [inline cobbler.spec:41]: Wrap these `undefine` statements in `0%{?suse_version} && 0%{?suse_version} < 1550` conditional, perhaps?

**Comment by watologo1** [comment]
Integrated in another request (for now without version condition, but this will come as soon as it gets more mature in SUSE build service and as soon as I have more time to try on this).
https://github.com/cobbler/cobbler/pull/2323

**Pull request closed without merging**